File: Algorithms_Data_Structures/Employee_Management_System/EmployeeTest.java
public class EmployeeTest {
    public static void main(String[] args) {
        EmployeeManager manager = new EmployeeManager(5);

        manager.addEmployee(new Employee("E101", "John", "Manager", 75000));
        manager.addEmployee(new Employee("E102", "Sarah", "Developer", 68000));
        manager.addEmployee(new Employee("E103", "Mike", "Designer", 55000));

        manager.traverseEmployees();

        System.out.println("\nSearching for E102:");
        System.out.println(manager.searchEmployee("E102"));

        System.out.println();
        manager.deleteEmployee("E102");
        manager.traverseEmployees();
    }
}

File: Week_1/Algorithms_Data_Structures/Employee_Management_System/EmployeeTest.cs
using System;

public class EmployeeTest {
    public static void Main(string[] args) {
        EmployeeManager manager = new EmployeeManager(5);

        manager.AddEmployee(new Employee("E101", "John", "Manager", 75000));
        manager.AddEmployee(new Employee("E102", "Sarah", "Developer", 68000));
        manager.AddEmployee(new Employee("E103", "Mike", "Designer", 55000));

        manager.TraverseEmployees();

        Console.WriteLine("\nSearching for E102:");
        Console.WriteLine(manager.SearchEmployee("E102"));

        Console.WriteLine();
        manager.DeleteEmployee("E102");
        manager.TraverseEmployees();
    }
}